ports/stm32/main.c

```diff
@@ -426,6 +428,9 @@ void stm32_main(uint32_t reset_mode) {
     #endif
     systick_enable_dispatch(SYSTICK_DISPATCH_LWIP, mod_network_lwip_poll_wrapper);
     #endif
+    #if defined(STM32WB)
+    rfcore_init();
```

Comment:
Why was this moved down here?  It should be fine here, just curious.

Reply:
Ah, leftover from an earlier trial when I thought the startup timing of this might be a key impact... yeah I don't have a reason to keep it moved.

Reply:
OK, then let's keep it where it was originally